src/Vendr.Contrib.Reviews/Web/Controllers/ReviewApiController.cs

```diff
@@ -42,7 +43,7 @@ public class ReviewApiController : UmbracoAuthorizedApiController
         public ReviewApiController(
             IReviewService reviewService,
             ILocalizedTextService textService,
-            IProductAdapter productAdapter)
+            IProductAdapter productAdapter):base()
```

Comment:
Any reason it needs to inherits from `base` here?

Reply:
No :) I think I must have left that in from some testing when I was doing the dev work - I've removed it now.